boards/arm/stm32/nucleo-l152re/src/stm32_ili93418b.c

```diff
@@ -0,0 +1,720 @@
+/****************************************************************************
+ * boards/arm/stm32/nucleo-l152re/src/stm32_ili93418b.c
+ *
+ * Driver for the ILI9341 Single Chip LCD driver connected
+ * via 8080-I 8-bit Series Parallel Interface
+ *
+ *   Author: Chunyang Liu <yjdwbj@gmail.com>
```

Comment:
Please remove the author from header file, here is some dissussion:
https://github.com/apache/incubator-nuttx/pull/2375